Gordon360/Documentation/Gordon360.xml

```diff
@@ -284,12 +284,16 @@
         </member>
         <member name="M:Gordon360.Controllers.Api.NewsController.GetNewsPersonalUnapproved">
             Call the service that gets all unapproved student news entries (by a particular user)
-            not yet expired, filtering
-            out the expired by comparing 2 weeks past date entered to current date
+            not yet expired, filtering out the expired news
```

Comment:
Might be a good idea to leave in a note about what classifies as "expired news" unless that is listed somewhere else